README.md

```diff
@@ -11,9 +11,17 @@ Please read the wiki for more details about checks already implemented.
 
 ## Dependencies
 
+### Boost
+
 The dependencies are the Boost library core, and its submodules: Boost.filesystem, Boost.program_options, Boost.regex and Boost.system.
 If you are using Ubuntu, the required packages names will be `libboost-dev`, `libboost-filesystem-dev`, `libboost-program-options-dev` and `libboost-regex-dev`.
 
+### ODB
+
+You will need to download the odb compiler, the odb common runtime library, and the sqlite database runtime library from this page: http://codesynthesis.com/products/odb/download.xhtml
+
+To install the odb compiler, the easiest way is to download the `.deb` or `.rpm` version, in order to be installed automatically with `dpkg`. Both the odb runtime and sqlite libraries can be installed with the usual `./configure && make && sudo make install`.
```

Comment:
We should be more specific about what refers to the package and what to the manual installation, otherwise I see non-technical users trying to run `configure` on a .deb... Maybe something like:

"If not using these type of packages, both the odb runtime and sqlite libraries can be manually build and installed"